lib/sentry/logger_handler.ex

```diff
@@ -86,6 +86,20 @@ defmodule Sentry.LoggerHandler do
       where it starts using `result: :sync` to block until the event is sent. If you always
       want to use sync mode, set this option to `0`. This option effectively implements
       **overload protection**.
+
+      `sync_threshold` and `discard_threshold` cannot be used together. To disable this option,
+      set it to `nil`.
+      """
+    ],
+    discard_threshold: [
+      type: {:or, [nil, :non_neg_integer]},
+      default: nil,
+      doc: """
+      *since v10.8.2* - The number of queued events after which this handler will start
+      to discard events. This option effectively implements **load shedding**.
+
+      `discard_threshold` and `sync_threshold` cannot be used together. To disable this option,
+      set it to `nil`.
```

Comment:
```suggestion
      The number of queued events after which this handler will start
      to **discard** events. This option effectively implements **load shedding**.
      
      *Available since v10.8.2*.

      `:discard_threshold` and `:sync_threshold` cannot be used together. To disable this option,
      set it to `nil`.
```